docs/pages/admin-guides/management/guides/scim-sailpoint.mdx

```diff
@@ -0,0 +1,175 @@
+---
+title: SCIM Sailpoint Integration
+description: How to manage Access List membership via SCIM SailPoint integration
+
+---
+
+The SCIM integration between **SailPoint** and **Teleport** enables automated
+synchronizing SailPoint group memberships with
+Teleport Access Lists. This integration supports centralized identity governance in
+SailPoint while Teleport enforces fine-grained access controls.
+
+User permissions in Teleport are defined through Access Lists. While role definitions
+live in Teleport, **group membership is dynamically managed** by SailPoint SCIM group
+assignments. This ensures users have up-to-date access aligned with organizational
+policies.
+
+## Prerequisites
+- Teleport Enterprise v(=teleport.version=)
+- Teleport Terraform provider v(=teleport.version=)
```

Comment:
I think we should link to the section of the docs that explains how to set up the provider: https://goteleport.com/docs/admin-guides/infrastructure-as-code/terraform-provider/